### PR TITLE
Add C# support classes to better integrate the .NET client with the Mono version of Godot…

### LINKED ISSUE #87: Add C# HTML5 export with the .NET client
Hello,

Running the sample `var session = await client.AuthenticateEmailAsync(email, password)`  doesn't work in HTML5 export. Works fine in the Windows one. Something similar to the Unity Client needs to be done, making use of Godot's WebSocket API, E.G:
```
private Godot.WebSocketClient ws;
public override void _Ready()
{
    ws = new Godot.WebSocketClient();
    GD.Print(ws.ConnectToUrl("ws://127.0.0.1:8080/ws-chat/123?name=Hostname", new string[1]{"my-protocol"}, false));
}
```

Relevant thread:
https://gitter.im/heroiclabs/nakama?at=61d6108e82a4667b255f8393

Thank you!

### PR DESCRIPTION
**Pull request by dsnopek**
… allowing HTML5 exports to work.

Fixes #87

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,6 +118,62 @@ func _on_socket_error(err):
...
+	try {
+		session = await client.AuthenticateDeviceAsync(OS.GetUniqueId(), "TestUser", true);
+	}
+	catch (ApiResponseException e) {
+		GD.PrintErr(e.ToString());
+		return;
+	}
+
+	var websocket_adaptor = new GodotWebSocketAdapter();
+	// Like the HTTP adaptor, it's a Node, so it needs to be added to the scene tree.
```
Comment (lugehorsam) [inline README.md:167]: tiny spelling fix: *adapter

File: README.md [hunk 2]
```diff
@@ -118,6 +118,62 @@ func _on_socket_error(err):
...
+	// Consider putting this in an autoload singleton so it won't go away unexpectedly.
+	AddChild(websocket_adaptor);
+
+	// Pass in the 'websocket_adaptor' as the last argument.
+	var socket = Socket.From(client, websocket_adaptor);
+```
+
+**Note:** _Out-of-the-box the Nakama .NET client will work fine with desktop builds of your game! However, it won't work with HTML5 builds, unless you use the `GodotHttpAdapter` and `GodotWebSocketAdapter` classes._
```
Comment (lugehorsam) [inline README.md:175]: *The out-of-the-box Nakama .NET...

**Comment by lugehorsam** [comment]
👍 two minor fixes, otherwise looks great

File: README.md (hunk 2 above)
Comment (dsnopek) [reply README.md]: Fixed!

File: README.md (hunk 1 above)
Comment (dsnopek) [reply README.md]: Fixed!

**Comment by dsnopek** [comment]
@lugehorsam Thanks, I've fixed those two things!

**Review by lugehorsam — APPROVED** [review]

**Pull request merged**